util/include/util/dbutil.h

```diff
@@ -5,6 +5,7 @@
 #include <string>
 
 #include <odb/database.hxx>
+#include <odb/sqlite/exceptions.hxx>
```

Comment:
This include should be guarded with `DATABASE_SQLITE`. There are no guarantees for this header to exist, if someone is only compiling for Postgres, they need not have the SQLite connector library's headers.

Reply:
Must have gotten there accidentally, because all SQLite specific things are in the `.cpp` file. Removed in 64cb407581662e78675baf916a95dba2b765b3e4.